NSS-Day-Cohort-55/automated-world-granny-mashinini-fires
# resized logo

# Description
resized logo

## Type of change

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions
check to make sure logo resize looks alright - will probably need to be positioned with flexbox


# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
